.github/workflows/main.yml

```diff
@@ -135,7 +135,8 @@ jobs:
           cd src
           bazel test -c dbg  \
             --run_under="valgrind --leak-check=yes --undef-value-errors=no --error-exitcode=1"   \
-            -- //firmware/...                 
+            -- //firmware/...   \
+	    -//firmware/boards/estop_arduino_uno/... # Arduino requires setup_firmware              
```

Comment:
this looks weird, could be messing up CI

Reply:
dang tabs, CI now working